README.md

```diff
@@ -161,6 +161,29 @@ The following settings from the previous section are also supported:
 - `hostname`
 - `log-raw-metric`
 
+Running as a Lambda Extension
+-----------------------------
+Gostatsd can be run as a lambda extension in forwarder mode. The metrics are flushed at the end of each lambda invocation
```

Comment:
I would highlight that this is still considered experimental / alpha and will be GA in an upcoming TBD future releas.es